src/SFA.DAS.AssessorService.Database/Tables/Standards.sql

```diff
@@ -18,5 +18,6 @@
     [VersionLatestEndDate] DATETIME NULL, 
     [VersionApprovedForDelivery] DATETIME NULL, 
     [ProposedTypicalDuration] INT NOT NULL, 
-    [ProposedMaxFunding] INT NOT NULL
+    [ProposedMaxFunding] INT NOT NULL,
+	[EPAChanged] BIT NOT NULL
```

Comment:
Just need to set a default of 0 so that the publish works and gets overridden the first time the import runs